index.bs

```diff
@@ -134,10 +134,13 @@ Issue(whatwg/fs#101): Make these checks no longer associated with an entry.
   1. If |status|.{{PermissionStatus/state}} is not {{PermissionState/"prompt"}}, abort.
   1. Let |settings| be |desc|.{{FileSystemPermissionDescriptor/handle}}'s [=relevant settings object=].
   1. Let |global| be |settings|'s [=environment settings object/global object=].
-  1. If |global| is not a {{Window}}, throw a {{SecurityError}}.
-  1. If |global| does not have [=transient activation=], throw a {{SecurityError}}.
-  1. If |settings|'s [=environment settings object/origin=] is not [=same origin=] with |settings|'s [=top-level origin=],
-     throw a {{SecurityError}}.
+  1. If |global| is not a {{Window}},
+     throw a "{{SecurityError}}" {{DOMException}}.
```

Comment:
not new but: try to write "if ..., then ..." (i.e. add "then" here and elsewhere)